ios/FluentUI/Avatar/AvatarTokens.swift

```diff
@@ -69,13 +69,13 @@ open class AvatarTokens: ControlTokens {
     open var ringDefaultColor: DynamicColor {
         switch style {
         case .default, .group:
-            return globalTokens.brandColors[.tint10]
+            return aliasTokens.strokeColors[.brandStroke1]
```

Comment:
I think you're missing #1032 , and these should be `aliasTokens.colors[.brandStroke1]` etc